lib/transformations/resolve/resolve.js

```diff
@@ -0,0 +1,46 @@
+// Press ctrl+space for code completion
+export default function transformer(file, api) {
```

Comment:
@DanielaValero - This is a WIP for me. Lets setup some working stuff before we open this to contributors. I'm hesitant to give a timeline for this as I can't commit fixed time everyday. But I have a plan to explore a couple more of these transformers and get it working. Then, I ll be able to abstract out common scenarios by writing utils. Then I will put all of these transformers under `webpack --migrate` at which point we can open this up to future contributors. I will update all of this in #29 and if needed create another issue. For speed and brevity, I prefer putting up PRs of whatever work I complete as long as it does not break existing functionality.